docs/ape17.rst

```diff
@@ -0,0 +1,533 @@
+       from ConfigParser import ConfigParser
+   except ImportError:
+       from configparser import ConfigParser
+
+you should change it to:
+
+.. code:: python
+
+   try:
+       from sphinx_astropy.conf.v1 import *  # noqa
+   except ImportError:
+       print('ERROR: the documentation requires the sphinx-astropy package to be installed')
+       sys.exit(1)
+
+   # Get configuration information from setup.cfg
+   from configparser import ConfigParser
+   conf = ConfigParser()
+
+Find and replace any instances of ``package_name`` in the file with
+``name``.
+
+Step 12 - Setting up tox
+------------------------
+
+`tox <https://tox.readthedocs.io/en/latest/>`__ is a tool for automating
+commands, which is well suited to e.g. running tests for your package or
+building the documentation. One of the benefits of using tox is that it
+will (by default) create a source distribution for your package and
+install it into a virtual environment before running tests or building
+docs, which means that it will be a good test of whether e.g. you have
+declared the package data correctly.
+
+As a starting point, copy over the ``tox.ini`` file generated in :ref:`Step 0 <step0>`.
+You can always then customize it if needed (although it should work out
+of the box).
+
+Once you have done this you should be able to do the following:
+
+Run tests with minimal dependencies::
+
+   tox -e test
+
+Run tests with astropy LTS and Numpy 1.16::
+
+   tox -e test-astropylts-numpy116
+
+Run tests with all optional dependencies::
+
+   tox -e test-alldeps
+
+Run tests with minimal dependencies and the latest developer version of
+numpy and astropy::
+
+   tox -e test-devdeps
+
+Build the documentation::
+
+   tox -e build_docs
+
+Run code style checks on your code::
+
+   tox -e codestyle
+
+The ``{posargs}`` corresponds to arguments passed to ``tox`` after a
+``--`` separator - for example to make pytest verbose in a ``test``
+environment, you can do::
+
+   tox -e test -- -v
+
+Step 13 - Updating your Continuous Integration
+----------------------------------------------
+
+This step will depend on what continuous integration services you use. Broadly
+speaking, unless there are dependencies you need that can only be installed with
+conda, you should no longer need to use ci-helpers to install these. The
+recommended approach is to use the tox file to set up the different
+configurations you want to use, and to then keep the CI configuration as simple
+as possible.
+
+If you use Travis CI, a good place to start is the ``.travis.yml`` file
+generated in :ref:`Step 0 <step0>`, and you can then see if any previous customizations you had
+made need to be copied over. This file shows how one can configure Travis to use
+tox, optionally using conda via ci-helpers to set up Python on MacOS X and
+Windows.
+
+Step 14 - Update ReadTheDocs configuration
+------------------------------------------
+
+With the set-up described in this migration guide, you should be able to
```

Comment:
Add a link to https://docs.readthedocs.io/en/stable/config-file/v2.html

Reply:
Done